### PR TITLE
Add translation.md for Weblate.org

### LINKED ISSUE #29548: Provide instructions on how to use Weblate for translations
### Description

There should be a file with instructions on how to translate using Weblate

### PR DESCRIPTION
**Pull request by robson90**
Closes #29548

### CODE REVIEW

**Comment by jonkoops** [comment]
Thanks for the PR. Could you link an issue to the PR so that we can track this one? Please also make sure to link any previous discussion in said issue.

File: docs/translation.md
Comment (jonkoops) [inline docs/translation.md:31]: Let's remove the number here, as it is likely to get out of date.
```suggestion
Please note that while we aspire to support more languages in the future, our current focus is on these languages. We rely on volunteers like you to take the initiative in steering the translation efforts for additional languages. If you're interested in contributing translations for a language not listed above, we encourage you to register on Weblate and reach out to join the community to kickstart the translation process for your language.
```

File: docs/translation.md
Comment (jonkoops) [inline docs/translation.md:37]:
```suggestion
Dutch: [[Jon Koops](https://github.com/jonkoops)]
```

File: docs/translation.md
Comment (jonkoops) [inline docs/translation.md:33]: I think rather than 'coordinators' we should call them 'maintainers' of languages, where those contributing could just be named 'contributors'

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: docs/translation.md
Comment (ahus1) [reply docs/translation.md:33]: will do in next commit

File: docs/translation.md
Comment (ahus1) [reply docs/translation.md:9]: There used to be a Japanese translation of the docs, see https://keycloak-documentation.openstandia.jp/ ... it seems to lack behind unfortunately. As a matter of completeness, I'll add it here as well.

File: docs/translation.md
Comment (ahus1) [reply docs/translation.md:8]: I'm a fan of a heading that explains the "why", so I tend to keep it. I removed one sentence from the first paragraph. Let me know what you think.

**Comment by ahus1** [comment]
@robson90 - I pushed some changes. The major change is that I updated "language" with "translation", and then removed  English from the list, as that is not a translation, but the source. 

IMHO only those translations are in scope for this document, and not the source language.

File: docs/translation.md
Comment (robson90) [inline docs/translation.md:26]: By reading again, I would also change this to:

```suggestion
- Join the Keycloak translation team on Weblate to start contributing translations.
```

**Comment by robson90** [comment]
> @robson90 - I pushed some changes. The major change is that I updated "language" with "translation", and then removed English from the list, as that is not a translation, but the source.
> 
> IMHO only those translations are in scope for this document, and not the source language.

@ahus1 Im fine with he changes, thanks for taking this over! 🎉

Could you please check if it is possible to disable the language request on Weblate ? Then we have only one point of interaction.

**Comment by ahus1** [comment]
> Could you please check if it is possible to disable the language request on Weblate ? Then we have only one point of interaction.

I found that I can switch in each component, in the settings on the tab "Files", I can choose for "Add new translation" the option "Point to translation instructions URL". 

Once this PR is then merged, we can update the "Translation instructions" in the settings for the Weblate Keycloak project to this new translation.md file.

**Comment by ahus1** [comment]
@jonkoops - could you please re-review the latest changes if you are ok to merge this PR as a first version? Once it is merged, I'll update the translation instructions to point to the new file. Once that is in place, IMHO we are ready to add one more language.

File: docs/translation.md
Comment (jonkoops) [inline docs/translation.md:16]: Perhaps good to mention that this is not maintained by the Keycloak team itself, but a third party.

File: docs/translation.md
Comment (jonkoops) [inline docs/translation.md:49]: Perhaps we could also create a nomination proccess here, if someone wants to volunteer to be a maintainer for the translations of a specific language on Weblate, we can ask them to open a PR to add a new language here and add their name as a maintainer. WDYT?

File: docs/translation.md
Comment (jonkoops) [inline docs/translation.md:1]: Perhaps we should link this document from the contribution guide as well, so that folks end up here when they are looking to contribute.

**Review by jonkoops — APPROVED** [review]
LGTM, just a couple of comments, but feel free to merge.

File: docs/translation.md
Comment (ahus1) [reply docs/translation.md:1]: Yes - this is the next step as discussed with @robson90

File: docs/translation.md
Comment (ahus1) [reply docs/translation.md:49]: Sounds like a follow-up PR to me. For the time being, the "If you're interested in contributing translations for a language not listed above, add a comment" should work.

**Review by ahus1 — APPROVED** [review]

**Comment by ahus1** [comment]
Thank you for everyone who contributed. This is now linked in the project description on Weblate: https://hosted.weblate.org/projects/keycloak/#information

**Pull request merged**
